pallets/teerex/ias-verify/src/lib.rs

```diff
@@ -133,6 +139,70 @@ pub struct SgxReport {
 	pub build_mode: SgxBuildMode,
 }
 
+#[derive(Encode, Decode, Clone, TypeInfo, sp_core::RuntimeDebug, Default)]
+pub struct SgxQuoteInputs {
+	pub spid: [u8; 16],
+	pub nonce: [u8; 16],
+
+	// the revocation list
+	pub sig_rl: Vec<u8>,
+}
+
+#[derive(Encode, Decode, Clone, TypeInfo, Default, sp_core::RuntimeDebug)]
+pub struct SgxQuoteAdd {
+	pub quote_inputs: SgxQuoteInputs,
+}
+
+#[derive(Encode, Decode, Clone, TypeInfo, sp_core::RuntimeDebug)]
+pub struct SgxTargetInfo {
+	// target info
+	pub mr_enclave: [u8; 32],
+	pub attributes: SGXAttributes,
+	pub reserved1: [u8; SGX_REPORT_BODY_RESERVED1_BYTES],
+	pub config_svn: u16,
+	pub misc_select: [u8; 4],
+	pub reserved2: [u8; SGX_REPORT_BODY_RESERVED2_BYTES],
+	pub config_id: [u8; 64],
+	pub reserved3: [u8; SGX_REPORT_BODY_RESERVED3_BYTES],
+}
+impl Default for SgxTargetInfo {
+	fn default() -> Self {
+		SgxTargetInfo {
+			mr_enclave: [0_u8; 32],
+			attributes: SGXAttributes { flags: 0_u64, xfrm: 0_u64 },
+			reserved1: [0_u8; SGX_REPORT_BODY_RESERVED1_BYTES],
+			config_svn: 0_u16,
+			misc_select: [0_u8; 4],
+			reserved2: [0_u8; SGX_REPORT_BODY_RESERVED2_BYTES],
+			config_id: [0_u8; 64],
+			reserved3: [0_u8; SGX_REPORT_BODY_RESERVED3_BYTES],
+		}
+	}
+}
+
+#[derive(Encode, Decode, Clone, TypeInfo, sp_core::RuntimeDebug)]
+pub struct SgxReportInputs {
+	pub target_info: SgxTargetInfo,
+	pub report_data: [u8; SGX_REPORT_DATA_SIZE],
+}
+impl Default for SgxReportInputs {
+	fn default() -> Self {
+		SgxReportInputs {
+			target_info: SgxTargetInfo::default(),
+			report_data: [0_u8; SGX_REPORT_DATA_SIZE],
+		}
+	}
+}
```

Comment:
just add `Default` derive for `SgxReportInputs`

Reply:
the trait `Default` is not support `[u8; SGX_REPORT_DATA_SIZE]`, SGX_REPORT_DATA_SIZE = 64